docs/learn/background/Gap Between Rollups.md

```
@@ -0,0 +1,40 @@
+
+* The latest Arbitrum block has no knowledge of Ethereum blocks beyond block 75, which might include blocks that other rollups like Base and Optimism are building their latest blocks on.  
+* Conversely, Base lacks information about the latest blocks of Optimism, but it does have knowledge of Arbitrum's latest L1 orgin since it has already processed Ethereum block 75\.
+
+This means that depending on their L1 origins, different L2s might be operating with different views of the Ethereum chain. They are either "in the past", “in the future” or "in the same time" relative to one another depending on where they are anchored in the L1 history.
+
+## Uniting Ethereum Blockspace 
+
+To ensure that rollups communicating within the finality gap are on the correct fork of Ethereum, Polymer Hub operates as a rollup on Ethereum itself. Being a rollup allows it to be directly connected to both Ethereum and other rollups, ensuring consistency and alignment across the network.
+
+![image (29)](https://github.com/user-attachments/assets/41881778-2fe7-41e2-99ef-4e8cf15413d0)
+- Rollups A, B and C will be allowed to communicate with each other in real time.
```

Markdownlint

Expected: asterisk; Actual: dash
Unordered list style

(MD004, ul-style)